holidays/constants.py

```diff
@@ -42,4 +43,5 @@
     ISLAMIC,
     PUBLIC,
     SCHOOL,
+    WORKDAY,
```

Comment:
The category list keeps growing. It has to be documented when/how each of the categories should be used. Ideally with some use case examples.

Reply:
I'll work w/ you on that soon, filed #1348 for now.